frontend/src/metabase/actions/components/ActionViz/Action.unit.spec.tsx

```diff
@@ -65,7 +67,20 @@ async function setup(options?: Partial<ActionProps>) {
 }
 
 async function setupActionWrapper(options?: Partial<ActionProps>) {
-  return renderWithProviders(<Action {...defaultProps} {...options} />);
+  return renderWithProviders(<Action {...defaultProps} {...options} />, {
+    withSampleDatabase: true,
+    storeInitialState: {
+      entities: {
+        databases: {
+          1: createMockDatabase({ id: 1 }),
+          2: createMockDatabase({
+            id: 2,
+            settings: { "database-enable-actions": true },
+          }),
+        },
+      },
+    },
```

Comment:
Can we please refactor the `Action` component to fetch databases it needs via entity loaders vs. relying on them being already loaded? If we change something about the data flow in parent components, it can break without signal

Reply:
https://github.com/metabase/metabase/pull/28418